src/libmongoc/tests/test-mongoc-cache.c

```diff
@@ -41,6 +41,7 @@ ping ()
 
    uri = bson_strdup_printf ("mongodb://localhost/?tls=true&tlsCAFile=%s",
                              ca_file);
+   /* TODO 3954: Replace with test_framework_client_new */
```

Comment:
Isn't that this ticket?

Reply:
That todo was obsolete: the code wouldn't compile with the new method, which is something @samantharitter also experienced. I've left this as-is for the time being.